kornia/color/lab.py

```diff
@@ -194,14 +191,13 @@ def __init__(self) -> None:
 
         self.xyz_to_rgb = torch.tensor([
             #     r           g          b
-            [ 3.24048134, -0.96925495,  0.05564664], # x
-            [-1.53715152,  1.87599, -0.20404134], # y
-            [-0.49853633,  0.04155593,  1.05731107], # z
+            [3.24048134, -0.96925495, 0.05564664],  # x
+            [-1.53715152, 1.87599, -0.20404134],  # y
+            [-0.49853633, 0.04155593, 1.05731107],  # z
```

Comment:
check this out: https://kornia.readthedocs.io/en/latest/color.html#kornia.color.xyz_to_rgb